scipy/sparse/tests/test_coo.py

```diff
@@ -229,15 +229,16 @@ def test_eliminate_zeros():
     assert arr1d.nnz == 1
     assert arr1d.count_nonzero() == 1
     assert np.array_equal(arr1d.toarray(), np.array([0, 1]))
-    assert np.array_equal(arr1d.row, np.array([1]))
+    assert np.array_equal(arr1d.col, np.array([1]))
+    assert np.array_equal(arr1d.row, np.array([0]))
 
 
 def test_1d_add_dense():
     den_a = np.array([0, -2, -3, 0])
     den_b = np.array([0, 1, 2, 3])
     exp = den_a + den_b
     res = coo_array(den_a) + den_b
-    assert type(res) == type(exp)
+    assert isinstance(res, np.ndarray)
```

Comment:
Why change this?

Reply:
I can't always keep in my head what type the numpy version creates. Testing equality with the dense version doesn't say what the result should be -- only that `res` matches `exp`. As I look at this again, I guess the `isinstance` version would pass even if one was a subclass of the other type. But I hadn't considered that before now.

I'm fine with going back to the original, but would like there to be some way to indicate what the type should be -- not just that they are equal. Perhaps: `type(res) == type(exp) == np.ndarray`?  But that feels like it is testing numpy (the `exp` part of the test).  And I'm fine with whatever we do here. I can change it back to the original and work to learn the numpy result types better. :)